config/environment.rb

```diff
@@ -0,0 +1,8 @@
+# Load the Rails application.
+require_relative "application"
+require_relative "env"
+
+# Initialize the Rails application.
+ENV['DATABASE_PASSWORD'] = DATABASE_PASSWORD
+ENV['DATABASE_USERNAME'] = DATABASE_USERNAME
```

Comment:
**OPTIONAL** - These lines cause issues when I try setting up the database, I suggest removing them from your app, please fix this 🙏